dangerfile.js

```diff
@@ -0,0 +1,22 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { danger, warn, fail } from 'danger';
+
+const hasCHANGELOGChanges = danger.git.modified_files.some((filePath) => {
+  const srcFilePattern = /CHANGELOG.md/i;
+  return srcFilePattern.test(filePath);
+});
+
+const hasModifiedFiles = danger.git.modified_files.some((filePath) => {
+  const filePattern = /(src|scripts|config)/;
```

Comment:
I think we can limit this to src

Reply:
good catch, I just copy/pasted most of the boilerplate/config from the terra-toolkit repo

Reply:
This has landed in master as just using `src` https://github.com/cerner/terra-enzyme-intl/blob/master/dangerfile.js#L10